README.md

```diff
@@ -133,9 +133,9 @@ public class MainActivity extends AppCompatActivity {
     super.onCreate(savedInstanceState);
     setContentView(R.layout.activity_main);
     // Automatic User Authentication
-    Thanx.initialize(this, "USER_ACCESS_TOKEN", true);
+    Thanx.initialize(this, "USER_ACCESS_TOKEN", true, true, null);
```

Comment:
Providing a method signature would be helpful here. It's a little tough to tell what the options are, other than by looking at the kotlin example.

Reply:
The developer can see the signature from the IDE directly too:
** java **
![Screen Shot 2019-09-24 at 11 12 25 AM](https://user-images.githubusercontent.com/1187096/65538533-5b63dd00-debc-11e9-8dd5-36226d50290a.png)